src/index.js

```diff
@@ -63,6 +63,32 @@ export default (function create(defaults) {
 		return (url, data, config) => redaxios(url, Object.assign({ method, data }, config));
 	}
 
+	/**
+	 * Builds request url based on query string parameters
+	 * @private
+	 * @param {string} url
+	 * @param {object|URLSearchParams} [params]
+	 * @param {function} [serializer]
+	 * @returns {string}
+	 */
+	function buildUrl(url, params, serializer) {
+		if (!params) {
+			return url;
+		}
+
+		let serializedParams;
+		if (serializer) {
+			serializedParams = serializer(params);
+		} else if (params instanceof URLSearchParams) {
```

Comment:
We can shrink this a bunch by leveraging the fact that URLSearchParams accepts itself as an argument:

```js
const serializedParams = serializer ? serializer(params) : new URLSearchParams(params);
const divider = ~url.indexOf('?') ? '&' : '?';
return url + divider + serializedParams;
```